test/tpu/Dockerfile

```diff
@@ -0,0 +1,127 @@
+ARG python_version=3.8
```

Comment:
Can this be removed in favor of the experimental Dockerfile? Does it need any changes to support tests? https://github.com/pytorch/xla/blob/master/docker/experimental/Dockerfile

Reply:
Yeah I am using that dockerfile locally to test right now. This is an old push, the PR is still in draft

Reply:
Done